### ISSUE TITLE
Joplin app

### ISSUE THREAD

**Issue by sttrebo** [syncloud/platform#524]
I have been using [Joplin](https://joplinapp.org/) a while now instead of the Notes app.  it's a bit more powerful and has additional features that make it a nice Evernote alternative.  I was currious if there was a way to get it running on Syncloud.  Since Syncloud is Debian based, I found this [article](https://medium.com/@numbuk/joplin-on-raspberry-pi-raspbian-d046ec615e40).  Granted this talks about Rasberry Pi and I'm not sure of the differences, but would love to see this if it's doable.

thanks

**Comment by cyberb** [comment]
Looks good, if you want you can help us you can try porting it using our docs here:
https://github.com/syncloud/platform/wiki/App-porting-guide and I can help you.

**Comment by sttrebo** [comment]
i'll give it a shot, thanks for the help.  i did find that there is already a snap package available for it.  i'm assuming i can start with that and modify it?

**Comment by cyberb** [comment]
more or less, there are few differences like we do not use snapcraft high level build tool
Added a section about it:
https://github.com/syncloud/platform/wiki/App-porting-guide#syncloud-snap-package-building-differences

**Comment by cyberb** [comment]
This app does not really have a backend, it can save to various backends like nextcloud. Not sure what can we do, we help with automating backends with or without web installs.

If someone can find a good backend/server app for jopling we can reopen it.

**Comment by sttrebo** [comment]
sorry, never seem to have the time to try it myself.  but here is a link to their server app info.  
[Joplin server](https://github.com/laurent22/joplin/tree/dev/packages/server)
maybe worth a look?  if not, i've been using it with my nextcloud install and it's been pretty flawless.

**Comment by cyberb** [comment]
ok, there is a server: https://github.com/laurent22/joplin/tree/dev/packages/server
Google found few screenshots but it looks like it is very basic for syncing the notes, does not have any ldap looks like.

**Comment by sttrebo** [comment]
correct, there was some discussions a year ago or so to implement this [(](https://discourse.joplinapp.org/t/open-to-pr-for-joplin-server-sso-support/22289) but i think it was put on hold as being too much effort to do/maintain.  not sure if there's a workaround for this unfortunately.
